cmd/controller/app/options/options.go

```diff
@@ -150,6 +156,11 @@ const (
 	defaultMaxConcurrentChallenges   = 60
 
 	defaultPrometheusMetricsServerAddress = "0.0.0.0:9402"
+	defaultHealthzServerAddress           = "127.0.0.1:10257"
```

Comment:
I used the same default port as Kubernetes controller manager:

https://github.com/kubernetes/kubernetes/blob/806b30170c61a38fedd54cc9ede4cd6275a1ad3b/cmd/kube-controller-manager/app/options/options.go#L196

The kube-controller-manager binds to all interfaces `0.0.0.0` but I think that's because the same server is also used for metrics, so it needs to accept connections from clients other than the kubelet.

The kube-controller-manager has only the following flags: `--bind-address string` and `--secure-port int (default: 10257)`. See https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/

Here I have followed the existing pattern of starting a separate HTTP server for metrics, pprof and now healthz.
But that seems inefficient....especially if eventually each of these servers needs a TLS certificate.